.changeset/eleven-books-camp.md

```diff
@@ -0,0 +1,7 @@
+---
+"@liam-hq/erd-core": patch
+"@liam-hq/erd-web": patch
+"@liam-hq/cli": patch
+---
+
+✨feat: Introduce Liam ERD Web App
```

Comment:
Since the web version is not published in npm, versioning is not necessary. What do you think?
However, since there is a modification to erd-core in this PR, I think the changeset itself is necessary:

```suggestion
---
"@liam-hq/erd-core": patch
"@liam-hq/cli": patch
---

⚒️ Fixing CSS Modules error with toolbar
```

Reply:
Yes, it is my mistake!
I've fixed them🙏
e3f3f37ee15d04a223f56fe0e16235cfcf7c01c7